bears/general/FilenameBear.py

```diff
@@ -71,6 +77,18 @@ def run(self, filename, file,
                      file_naming_convention)
             return
 
+        if not filename_without_extension.startswith(prefix):
+            yield Result.from_values(
+                origin=self,
+                message='Filename does not use the {} prefix.'.format(prefix),
+                file=filename)
+
+        if not filename_without_extension.endswith(suffix):
+            yield Result.from_values(
+                origin=self,
+                message='Filename does not use the {} suffix.'.format(suffix),
+                file=filename)
```

Comment:
Like just appending the suffix^^
(and for the prefix: just appending the prefix ;D)